open_discussions/settings.py

```diff
@@ -556,7 +556,7 @@
     DEFAULT_FILE_STORAGE = "storages.backends.s3boto.S3BotoStorage"
 
 IMAGEKIT_SPEC_CACHEFILE_NAMER = "imagekit.cachefiles.namers.source_name_dot_hash"
-IMAGEKIT_CACHEFILE_DIR = get_string("IMAGEKIT_CACHEFILE_DIR", "")
+# IMAGEKIT_CACHEFILE_DIR = get_string("IMAGEKIT_CACHEFILE_DIR", "")
```

Comment:
Why is this commented out?

Reply:
Shouldn't be there, it's dupe of the line above, I'll remove it.

Reply:
Mistake, uncommented it